trailblazer/store/models.py

```diff
@@ -40,6 +40,7 @@ class User(Model):
     id = Column(types.Integer, primary_key=True)
     is_archived = Column(types.Boolean, default=False)
     name = Column(types.String(128))
+    refresh_token = Column(types.String(128))
```

Comment:
similar question here - do we know how long the token can be?

Reply:
Changed to text

Reply:
https://github.com/Clinical-Genomics/trailblazer/pull/402/commits/c701b25fa1ec3ccf464b4ab492bd9fcf815a5642